core/arch/arm/include/sm/optee_smc.h

```diff
@@ -353,6 +352,33 @@
 	OPTEE_SMC_FAST_CALL_VAL(OPTEE_SMC_FUNCID_ENABLE_SHM_CACHE)
 
 /*
+ * Release of secondary cores
+ *
+ * OP-TEE in secure world is in charge of the release process of secondary
+ * cores. The Rich OS issue the this request to ask OP-TEE to boot up the
+ * secondary cores, go through the OP-TEE per-core initialization, and then
+ * switch to the Non-secure world with the Rich OS provided entry address.
+ *
+ * Call register usage:
+ * a0	SMC Function ID, OPTEE_SMC_BOOT_SECONDARY
+ * a1   smp cpu index
+ * a2   NS entry address low
```

Comment:
To harmonize better with the other descriptions please rephrase as:
```
a2 Upper 32-bits of 64-bit NS entry address
a3 Lower 32-bits of 64-bit NS entry address
```
Any particular reason the lower bit was first?
Please add a comment about what can be expected of CPSR also.